Annotation of SCMS Metric (#110)
chaoss/wg-value: focus-areas/communal-value/social-currency-metric-system.md

# Social Currency Metric System (SCMS)

Question: How does one measure the value of community interactions and accurately gauge “reputation” of a community as evident from qualitative sentiment?

> *Note: This metric synthesizes several candidate atomic metrics that can be derived from trace data, and several candidate process oriented atomic metrics. Embedded footnotes annotate areas planned for later clarification, and questions for later resolution.*

Reviewer: I recommend we skip "atomic" as a concept until we centralize this terminology across working groups and differentiate them from "synthetic."

```suggestion
*Note: This metric synthesizes several other metrics that can be derived from trace data, and several process-oriented metrics. Embedded footnotes annotate areas planned for later clarification, and questions for later resolution.*
```

- [An example implementation in data studio](https://datastudio.google.com/open/1Z4EJ03898lZxm2NZVULaEoLS0bYqL79A) (data source)
- [An example implementation in google sheets](https://drive.google.com/open?id=1zi3JE0bwfEdRdc-wQEZn8GaB7sE8IvxeSeqvVywKnXw)
- [Implementation documentation](https://docs.google.com/document/d/1RlAedRBQbhq0oYMCB3VqdawOCZE2XT5R3teydjBZODM/edit#heading=h.8hyunaadfriq) (starts on page 33)

### Annotated Footnotes

> <sup>1</sup> CHAOSS metrics historically is to create standard, atomic definitions that can be used reliably across projects to support comparisons. This metric may evolve into a project in the future.

Reviewer: ```suggestion
<sup>1</sup> CHAOSS metrics historically is to create standard definitions that can be used reliably across projects to support comparisons. This metric may evolve into a project in the future.
```

> <sup>2</sup> What atomic metrics emerge from this description? Likely included are: 1. community trust, 2. transparency,  3. utility, 4. consistency, and 5. merit 

Reviewer: ```suggestion
<sup>2</sup> What metrics emerge from this description? Likely included are: 1. community trust, 2. transparency,  3. utility, 4. consistency, and 5. merit 
```
